xarray/backends/api.py

```diff
@@ -870,13 +872,27 @@ def open_mfdataset(
     """
     if isinstance(paths, str):
         if is_remote_uri(paths):
-            raise ValueError(
-                "cannot do wild-card matching for paths that are remote URLs: "
-                "{!r}. Instead, supply paths as an explicit list of strings.".format(
-                    paths
-                )
+            from fsspec import open_files
```

Comment:
Do you need to handle the case when fsspec is not installed?

Reply:
As it is, this would raise ImportError. Would it be better to have a try/except ImportError/raise from, with an extra sentence?
"To open fsspec-compatible [remote?] URLs, you must install fsspec"
Note that you might still get an error in this block if, for example, it's an s3 URL, but s3fs is not installed.

Reply:
I thought because L893 cannot be reached without fsspec.

Reply:
(I don't really mind - just because you mention backward compatibility - I am also fine to leave it as is)